README.md

```diff
@@ -5,86 +5,95 @@ Asymmetrik FHIR API Server
 
-Familiarity of the FHIR specification and whatever database containing the resources will assist in querying data and returning them in a format conforming to specification.
+1. Install [Node.js](https://nodejs.org/en/) latest version (7.7+, **but** not 8.5, see [Attention](#attention)) and NPM included with Nodejs.
+2. Optional: Install [Yarn](https://yarnpkg.com/lang/en/docs/install) latest version.
+
+Understanding of JavaScript [promise](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise) and familiarity of the FHIR specification would be helpful.
 
 ## Getting Started
-Install `node-fhir-server-core` as followed:
 
-```shell
-# For yarn users
-yarn add git+https://github.com/Asymmetrik/node-fhir-server-core.git
+1. Install `node-fhir-server-core` initial modules with either NPM or Yarn:
 
-# For npm users
+```shell
 npm install --save git+https://github.com/Asymmetrik/node-fhir-server-core.git
+#OR
+yarn add git+https://github.com/Asymmetrik/node-fhir-server-core.git
 ```
 
-Once installed, the server can be required in the application.
 
+
+2. File `node-fhir-server-core/src/runner.js` allows custom authorization server which should be incorporated separately. The server portion can be included as followed:
+ 
 ```javascript
 const { VERSIONS } = require('@asymmetrik/node-fhir-server-core/src/constants');
 const fhirServerCore = require('@asymmetrik/node-fhir-server-core');
 
-/**
-* The configurations below are required and the server
-* will not run unless you have specified a server.port
-* and at least one valid profile configuration
-*/
+// Server requires server.port and at least one profile
+
 const config = {
-	server: { port: 3000 },
-	profiles: {
-		patient: {
-			service: path.resolve('./profiles/patient/patient.service'),
-			versions: [ VERSIONS.STU3 ]
-		}
+  server: { port: 3000 },
+  profiles: {
+    patient: {
+  	  service: path.resolve('./profiles/patient/patient.service'),
+	  versions: [ VERSIONS.STU3 ]
 	}
+  }
 };
 
-/**
-* fhirServerCore returns a promise so you can use async/await
-* if you like, or just use a normal promise like so:
-*
-* fhirServerCore(config)
-* 	.then(server => server.logger.info('success'))
-* 	.catch(err => console.error(err));
-*/
+
+// fhirServerCore returns a promise so async/await can be used
+
 let main = async function () {
-	// If the app fails to start, log the error
-	let server = await fhirServerCore(config).catch(console.error);
-	
-	if (server) {
-		server.logger.info(`FHIR Server it at http://localhost:${config.port}`);
-	}
+  let server = await fhirServerCore(config).catch(console.error);
+  if (server) {
+    server.logger.info(`FHIR Server it at http://localhost:${config.port}`);
+  }
 };
 
 main();
```

Comment:
I can help update this snippet, it is no longer used in this way and the recent updates to core have changed some requirements.